Source/items.cpp

```diff
@@ -1381,6 +1434,26 @@ void GetItemAttrs(int i, int idata, int lvl)
 			rndv = 5 * (currlevel + 16) + random(21, 10 * (currlevel + 16));
 		if (gnDifficulty == DIFF_HELL)
 			rndv = 5 * (currlevel + 32) + random(21, 10 * (currlevel + 32));
+#else
+	item[i]._iFlags = 0;
+	item[i]._iDamAcFlags = 0;
+
+	if (item[i]._iMiscId == IMISC_BOOK)
+		GetBookSpell(i, lvl);
+
+	if (item[i]._iMiscId == IMISC_OILOF) {

```

Comment:
probably should leave out the curly braces to match the `IMISC_BOOK` case from above.